WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -133,7 +133,7 @@
 
     private OnMagicLinkRequestInteraction mListener;
     private String mToken = "";
-    private boolean mInhibitMagicLogin;
+    private boolean mShouldReturnToPreviousActivity;
```

Comment:
@kwonye , can you elaborate on the rename? I know the commit message reads "...as it previously implied that it was non-magic links..." but not sure it implied that. Thanks!

Let me share some background on the naming of mInhibitMagicLogin:
`mInhibitMagicLogin` being true means the code should _prevent_ the magic-links flow from happening. `mInhibitMagicLogin` being false means the code is free to engage in magic-links or not, based on other conditions.
So, it's not a state variable for whether magic-links is actually in-effect or not. Perhaps we can tweak its name a bit, like rename it to "mPreventMagicLinks" if that makes it easier to understand.

Then again, "mShouldReturnToPreviousActivity" to me sounds like it might interfere somehow with the normal way Android handles activities. For example, if this fragment (parent activity anyway) was started via a `startActivityForResult` then the system _will_ return to the calling activity when this one finishes but, the `mShouldReturnToPreviousActivity` name seems to imply it might not.